resources/lib/uzg.py

```diff
@@ -42,7 +42,7 @@ def getItems(self, action, guid = None, productId = None, slug = None, text = No
         elif action == 'Zoeken':
             if text:
                 return self.queryItems.getItems(text)
-            return List()
+            return []
```

Comment:
```suggestion
            return list()
```

Dit zou ook kunnen

Reply:
list werkt niet in python 3.8. Windows versie word nog geleverd met python 3.8.
kan het wel anders fixxen, ga ik nog naar kijken

Reply:
`list` met kleine letter doet het waarschijnlijk wel in Python 3.8, dat is de built-in lijst. `typing.List` is alleen de type hint (/alias) die nieuw is in Python 3.9. Het verschil is dus tussen hoofdletter en kleine letter in dit PR.

Het toevoegen van type hints is sowieso best wel optioneel, dus als dat nu Windows support lastiger maakt...

Dus als je een lege lijst terug wil geven kan je `list()` doen, een lege list literal `[]`, maar niet `List()` met een hoofdletter. Dat is vooral wat ik in dit PR wilde fixen.